perfkitbenchmarker/virtual_machine.py

```diff
@@ -108,9 +97,8 @@ def __init__(self, vm_spec):
     """
     super(BaseVirtualMachine, self).__init__()
     with self._instance_counter_lock:
-      self.name = 'perfkit-%s-%d' % (FLAGS.run_uri, self._instance_counter)
+      self.name = 'pkb-%s-%d' % (FLAGS.run_uri, self._instance_counter)
```

Comment:
Could you make a separate PR for this? I'm used to looking for resources with a `perfkit-` prefix, others may be as well.